platform/android/api/jni_singleton.h

```diff
@@ -149,9 +149,18 @@ class JNISingleton : public Object {
 				ret = sarr;
 				env->DeleteLocalRef(arr);
 			} break;
+			case Variant::PACKED_FLOAT64_ARRAY: {
```

Comment:
For completion, we can also add the implementation for the `PACKED_INT64_ARRAY` variant type, using `long` instead of `int`.